lib/takeaway.rb

```diff
@@ -0,0 +1,92 @@
+require_relative 'messager'
+require_relative 'menu'
+
+class TakeAway
+  
+  attr_reader :read_menu, :menu, :orders, :messager      
+  
+  def initialize(menu = Menu.new, messager = Messager.new)
+    @menu_obj = menu
+    @messager = messager
+    @orders = []
+    @menu =  @menu_obj.menu
```

Comment:
The class shouldn't have access to `@menu_obj.menu` here.  The `menu` attribute now points to a Hash, but I would expect it to point the the `Menu` object I passed in if I'd overridden the default.